src/website/app/pages/Roadmap/roadmap.md

```diff
@@ -0,0 +1,31 @@
+# Roadmap
+
+If you want to see the types of work we're doing, what's been completed and what's left to do, you've come to the right place.
+
+## Current Sprint
+
+We're working in Agile sprints. Since all of our work is expressed in Github issues in our repo, we are able to publish our current progress and the issues that we're working on with Waffle.io. Feel free to check out what we're doing day-to-day if that's something that you find useful.
+
+<Button primary element='a' href='https://waffle.io/mineral-ui/mineral-ui'>View our Waffle Board</Button>
+
+## Past and Future Work
+
+Our day-to-day work always has an eye towards the bigger picture and the goals our team is trying to achieve. Below, you can get a high-level overview of what work we've done, what's in-progress, and what we're planning to do next.
```

Comment:
"Get a high-level overview of completed work, what's in progress, and what's planned below."